test/framework/type_checking.py

```diff
@@ -257,18 +257,19 @@ def test_to_dependency(self):
         lib_dict.update({'versionsuffix': ''})
 
         # to_dependency doesn't touch values of non-dict type
-        self.assertEqual(to_dependency(('foo', '1.3')), ('foo','1.3'))
-        self.assertEqual(to_dependency(('foo', '1.3', '-suff', ('GCC', '4.8.2'))), ('foo', '1.3', '-suff', ('GCC','4.8.2')))
+        self.assertEqual(to_dependency(('foo', '1.3')), ('foo', '1.3'))
+        expected = ('foo', '1.3', '-suff', ('GCC', '4.8.2'))
+        self.assertEqual(to_dependency(('foo', '1.3', '-suff', ('GCC', '4.8.2'))), expected)
```

Comment:
isn't this just self.assertEqual(to_dependency(expected), expecter) ?? And the same for all of them...
Or am I blind??

Reply:
No, you're right, we're checking here whether `to_dependency` leaves correct dependency specs as they are (see comment above)

Reply:
So why not use to_dep(exp), exp)? much easier to grasp.

Reply:
Good point, will change...